addons/website/views/snippets/s_cta_box.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_box" name="Box Call to Action">
+    <section class="s_cta_box pt80 pb80">
+        <div class="container">
+            <div class="s_card o_card_img_horizontal flex-lg-row-reverse card o_cc o_cc4" data-snippet="s_card" data-vcss="001" data-vxml="001" data-name="Card" style="--card-spacer-x: 64px; --card-spacer-y: 64px; border-width: 0px !important;">
+                <figure class="o_card_img_wrapper ratio ratio-1x1 mb-0">
+                    <img class="o_card_img card-img-top rounded-end" src="/web/image/website.s_cta_box_default_image"/>
```

Comment:
I forgot it in my last review ( :grimacing: ) but you should remove the `card-img-top` class (since we are horizontal) :+1: